packages/api/src/lib/granule-delete.d.ts

```diff
@@ -0,0 +1,2 @@
+export {};
+//# sourceMappingURL=granule-delete.d.ts.map
```

Comment:
What is this file?

Reply:
A compiled TS file that I definitely didn't mean to commit. I'll fix

Reply:
3136e699e4bd7953ba57aaaab21286b4383ca2f5